test/GivingCircle.Api.UnitTest/Controllers/TestBankAccountController.cs

```diff
@@ -0,0 +1,116 @@
+﻿using GivingCircle.Api.Controllers;
+using GivingCircle.Api.DataAccess.Fundraisers.Repositories;
+using GivingCircle.Api.Models;
+using GivingCircle.Api.Requests.FundraiserService;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using System;
+using Xunit;
+
+namespace GivingCircle.Api.UnitTest.Controllers
+{
+    public class TestBankAccountController
+    {
+        [Fact]
+        public async void TestAddBankAccountAsync()
+        {
+            // Given
+            var addBankAccountRequest = new AddBankAccountRequest
+            {
+                Account_Name = "Austin Nguyen",
+                Address = "55st hollywood blvd",
+                City = "Los Angeles",
+                State = "Kansas",
+                Zipcode = "10101",
+                Bank_Name = "Austin Bank",
+                Account_Num = "87870086",
+                Routing_Num = "402984494",
+                Account_Type = "Checkings",
+            };
+
+            var bankAccountRepositoryMock = new Mock<IBankAccountRepository>();
+            bankAccountRepositoryMock.Setup(r => r.AddBankAccount(It.IsAny<BankAccount>()))
+                .ReturnsAsync(true);
+
+            var loggerMock = new Mock<ILogger<BankAccountController>>();
+
+            var controllerMock = new BankAccountController(
+                loggerMock.Object,
+                bankAccountRepositoryMock.Object
+                );
+
+            // When
+            var result = await controllerMock.AddBankAccount(addBankAccountRequest) as StatusCodeResult;
+
+            // Then
+            Assert.Equal(StatusCodes.Status201Created, result.StatusCode);
+        }
+
+        [Fact]
+        public async void TestDeleteBankAccountAsync()
+        {
+            // Given
+            var Bank_Account_Id = Guid.NewGuid().ToString();
+
+            var bankAccountRepositoryMock = new Mock<IBankAccountRepository>();
+            bankAccountRepositoryMock.Setup(r => r.DeleteBankAccountAsync(It.IsAny<string>()))
```

Comment:
Instead of `It.IsAny<string>()` should you be passing the `Bank_Account_Id` variable?